pkg/nsx/services/subnetport/subnetport.go

```diff
@@ -387,6 +393,9 @@ func (service *SubnetPortService) AllocatePortFromSubnet(subnet *model.VpcSubnet
 		info.totalIp = totalIP - 4
 	}
 
+	if time.Since(info.exhausted) < IPReallocationTime {
```

Comment:
what does it mean?

Reply:
It checks if 2 minutes have passed since last time we received the IP exhausted error. We would like to avoid allocating more SubnetPorts on this IP-exhausted Subnet in 2 minutes.

Reply:
rename to exhaustedCheckTime?

Reply:
Updated, thanks!